examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<blockquote>The purpose of a <strong>constraint function</strong> is generally to model constraints and return whether those are still true</blockquote><p>The purpose of a constraint function is to allow performing an activity only while some specific constraint is true. We don't impose a constraint here, however, since (...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>